geoportailv3/static/js/draw/drawdirective.js

```diff
@@ -234,8 +257,8 @@ app.DrawController = function($scope, ngeoDecorateInteraction, ngeoLocation,
       function(evt) {
         goog.asserts.assertInstanceof(evt.element, ol.Feature);
         var feature = evt.element;
-        goog.events.listen(feature, goog.events.EventType.CHANGE,
-            this.onFeatureChange_, undefined, this);
+        goog.events.listen(feature, app.DrawEventType.PROPERTYMODIFYEND,
+            this.onFeaturePropertyChange_, undefined, this);
```

Comment:
I see. It makes sense. But in that case why do dispatch the event to the feature? Can't you use a more general event that is dispatched when the user clicks on the OK button?